MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2576,8 +2576,9 @@ private static int ScaleBitrate(int bitrate, string inputVideoCodec, string outp
             return 128000 * (outputAudioChannels ?? audioStream.Channels ?? 2);
         }
 
-        public string GetAudioVbrModeParam(string encoder, int bitratePerChannel)
+        public string GetAudioVbrModeParam(string encoder, int bitrate, int channels)
         {
+            var bitratePerChannel = bitrate / channels;
```

Comment:
```suggestion
            var bitratePerChannel = bitrate / Math.Max(channels, 1);
```

Would probably be caught upstream but if it is preventable might as well